cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -28,9 +28,9 @@ import (
 
 	hostpath "github.com/openebs/maya/pkg/hostpath/v1alpha1"
 
-	container "github.com/openebs/maya/pkg/kubernetes/container/v1alpha1"
-	pod "github.com/openebs/maya/pkg/kubernetes/pod/v1alpha1"
-	volume "github.com/openebs/maya/pkg/kubernetes/volume/v1alpha1"
+	"github.com/openebs/dynamic-localpv-provisioner/pkg/kubernetes/api/core/v1/container"
+	"github.com/openebs/dynamic-localpv-provisioner/pkg/kubernetes/api/core/v1/pod"
+	"github.com/openebs/dynamic-localpv-provisioner/pkg/kubernetes/api/core/v1/volume"
```

Comment:
Why is the package version changed here? Does it mean that it corresponds to `v1` version in upstream kubernetes?

Reply:
yes. to be in sync with the version in upstream kubernetes.